packages/crud/test/a11y.test.js

```diff
@@ -256,4 +256,56 @@ describe('a11y', () => {
       </vaadin-crud>
   `),
   );
+
+  ['aside', 'bottom'].forEach((editorPosition) => {
+    describe(`create item - editor position ${editorPosition}`, () => {
```

Comment:
It seems there is no need for two separate test suites because the setup remains the same.

Reply:
Done.